### DISCUSSION TITLE
Is there still Terminal Dev now?

### DISCUSSION THREAD

**Discussion by faq0** [microsoft/terminal#13278] [Q&A] [answered]
Is there still Windows Terminal Dev version (as that icon is on the readme.md file), or is it replaced by Terminal Preview?

**Comment by zadjii-msft** [comment, marked as answer]
Terminal Dev is the branding that's used when you build the Terminal locally. 

![image](https://user-images.githubusercontent.com/18356694/173211775-75a6b702-0bb3-4165-a77b-53fd2edede39.png)
